__tests__/app.test.js

```diff
@@ -0,0 +1,48 @@
+const connection = require("../db/connection");
+const app = require("../app");
+const seed = require("../db/seeds/seed");
+const {
+  categoryData,
+  commentData,
+  reviewData,
+  userData,
+} = require("../db/data/test-data/");
+const request = require("supertest");
+
+beforeEach(() => seed({ categoryData, commentData, reviewData, userData }));
+afterAll(() => connection.end());
+
+describe("GET /api/categories", () => {
+  test("200 response returns with an array", () => {
+    return request(app)
+      .get("/api/categories")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body).toBeInstanceOf(Array);
+      });
+  });
+  test("200 response returns array of objects with specific properties", () => {
+    return request(app)
+      .get("/api/categories")
+      .expect(200)
+      .then(({ body }) => {
+        console.log(body, "<<<in test file");
+        body.forEach((category) => {
+          expect(category).toEqual(
+            expect.objectContaining({
+              slug: expect.any(String),
+              description: expect.any(String),
+            })
+          );
+        });
+      });
+  });
```

Comment:
happy path 200 tests can go in the same test, but there is nothing wrong with the way you did it